### DISCUSSION TITLE
Xfinity Mobile

### DISCUSSION THREAD

**Discussion by MauricioDiaz** [jef/streetmerchant#1334] [Help]
Wondering if Xfinity Mobile is compatible? I put Verizon on the .env file as I know they use the Verizon towers. Maybe that will work?

**Comment by jef** [comment]
> XFinity Mobile: number@vtext.com

It sure looks like it off a Google lookup! If so, let me know. I'll add it in the notes.

**Reply by MauricioDiaz** [reply]
Hi Jef. I used verizon and it seems to work!

**Reply by jef** [reply]
Awesome. Thank you! I'll update the notes.
